chart-templates/crd-assume-ownership/templates/_helpers.tpl

```diff
@@ -0,0 +1,7 @@
+# Rancher
+
+{{- define "system_default_registry" -}}
+{{- if .Values.global.systemDefaultRegistry -}}
```

Comment:
Recently changed to `global.cattle.systemDefaultRegistry`

Reply:
will be updated in the latest commit